server/tests/routes/api/dev_place_test.py

```diff
@@ -0,0 +1,419 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import patch
+
+from server.routes.dev_place.types import Place
+from web_app import app
+
+SAMPLE_CHART_CONFIG = [{
+    "category": "Crime",
+    "titleId": "CHART_TITLE-Total_crime",
+    "title": "Total crime",
+    "description": "Total number of criminal incidents",
+    "statsVars": ["Count_CriminalActivities_CombinedCrime"],
+    "isOverview": True,
+    "relatedChart": {
+        "titleId": "CHART_TITLE-Crime_rate",
+        "title": "Crimes per 100,000 people",
+        "description": "Total number of criminal incidents per 100,000 people",
+        "scale": True,
+        "denominator": "Count_Person",
+        "scaling": 100000
+    }
+}, {
+    "category":
+        "Education",
+    "titleId":
+        "CHART_TITLE-Educational_attainment",
+    "title":
+        "Education attainment",
+    "description":
+        "Number of people who have attained various educational milestones, e.g. completed high school or have a bachelor's degree",
+    "statsVars": [
+        "Count_Person_EducationalAttainmentNoSchoolingCompleted",
+        "Count_Person_EducationalAttainmentRegularHighSchoolDiploma",
+        "Count_Person_EducationalAttainmentBachelorsDegree",
+        "Count_Person_EducationalAttainmentMastersDegree",
+        "Count_Person_EducationalAttainmentDoctorateDegree"
+    ],
+    "isOverview":
+        True,
+    "relatedChart": {
+        "titleId":
+            "CHART_TITLE-Educational_attainment_rate",
+        "title":
+            "Education attainment rate",
+        "description":
+            "Percentage of the adult population who have attained various educational milestones, e.g. completed high school or have a bachelor's degree",
+        "scale":
+            True,
+        "denominator":
+            "Count_Person_25OrMoreYears",
```

Comment:
consider adding another test with more denominators, and one without translations

Reply:
done!